Lesson26/app/feyza/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Image Gallery</title>
+  <link rel="stylesheet" type="text/css" href="./styles.css">
+</head>
+<body>
+  <h1>Image Gallery</h1>
+  <button id="fetch-button" class="button">Fetch Images</button>
+  <button id="clear-button" class="button">Clear</button>
+  <div id="image-container">
+    <label for="imageCount">Number of images to fetch:</label>
```

Comment:
Label and Input elements should not be inside image-container div. Move them above the div. Also, add `form` element that should wrap label and input elements.